config/bootstrap.php

```diff
@@ -84,6 +85,9 @@
         }
     }
 
+    $dotenv = new Dotenv();
+    $dotenv->loadEnv(_PS_ROOT_DIR_.'/.env');
```

Comment:
Watch out, if the file doesn't exist, it raises an exception

Reply:
loadEnv handles well environments
it will load in that order .env.{$env} .env.{$env}.local .env or fallback to .env.dist

But I can catch PathException for security ;)

Reply:
The dist file shouldn't be released. dist files are often configuration files that do not contain the real-world deploy-specific parameters and are there to help you get started with the application/framework faster.